rl_loop/update_resign_threshold.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Updates a flagfile containing a resign threshold flag
+
+Reads the bigtables defined by env vars in PROJECT,CBT_INSTANCE,CBT_TABLE to
+compute the 95 percentile of the bleakest-evaluations found in calibration games,
+then updates the flagfile on the default bucket path, resetting that value
+"""
+
+import sys
+import re
+import os
+import time
+from absl import app
+from absl import flags
+import numpy as np
+import tensorflow as tf
+
+sys.path.insert(0, '.')
+
+import bigtable_input
+import rl_loop.fsdb as fsdb
+
+
+FLAGS = flags.FLAGS
+RESIGN_FLAG_REGEX = re.compile(r'--resign_threshold=([-\d.]+)')
+
+def get_95_percentile_bleak(n_back=1000):
+    """Gets the 95th percentile of bleakest_eval from bigtable"""
+    end_game = int(bigtable_input._games_nr.latest_game_number())
+    start_game = end_game - n_back if end_game >= n_back else 0
+    moves = bigtable_input._games_nr.bleakest_moves(start_game, end_game)
+    evals = np.array([m[2] for m in moves])
```

Comment:
Now that I think of it, I should take the trouble to return a named tuple from `bleakest_moves`. I'll do it in another PR.

Reply:
SGTM